<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>devControl</class>
 <widget class="QMainWindow" name="devControl">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>480</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="0" column="0">
     <layout class="QVBoxLayout" name="verticalLayout" stretch="2,6,1">
      <property name="spacing">
       <number>0</number>
      </property>
      <item>
       <layout class="QGridLayout" name="gridLayout" columnstretch="1,2,2,2,2" columnminimumwidth="1,2,2,2,2">
        <item row="0" column="0">
         <widget class="DigitalClock" name="timewidget" native="true">
          <property name="maximumSize">
           <size>
            <width>240</width>
            <height>120</height>
           </size>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="AnimationButton" name="controlBtn" native="true">
          <property name="minimumSize">
           <size>
            <width>100</width>
            <height>120</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>100</width>
            <height>120</height>
           </size>
          </property>
         </widget>
        </item>
        <item row="0" column="2">
         <widget class="AnimationButton" name="setBtn" native="true">
          <property name="minimumSize">
           <size>
            <width>100</width>
            <height>120</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>100</width>
            <height>120</height>
           </size>
          </property>
         </widget>
        </item>
        <item row="0" column="3">
         <widget class="AnimationButton" name="offBtn" native="true">
          <property name="minimumSize">
           <size>
            <width>100</width>
            <height>120</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>100</width>
            <height>120</height>
           </size>
          </property>
         </widget>
        </item>
        <item row="0" column="4">
         <widget class="AnimationButton" name="backBtn" native="true">
          <property name="minimumSize">
           <size>
            <width>100</width>
            <height>120</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>100</width>
            <height>120</height>
           </size>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QWidget" name="widget" native="true">
        <property name="styleSheet">
         <string notr="true">background-color: rgb(115, 210, 22);

</string>
        </property>
        <layout class="QGridLayout" name="gridLayout_4">
         <item row="0" column="0">
          <layout class="QGridLayout" name="gridLayout_3">
           <item row="0" column="0">
            <widget class="LED" name="widget_3" native="true">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="minimumSize">
              <size>
               <width>120</width>
               <height>130</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>120</width>
               <height>130</height>
              </size>
             </property>
            </widget>
           </item>
           <item row="0" column="1">
            <widget class="LED" name="widget_4" native="true">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="minimumSize">
              <size>
               <width>120</width>
               <height>130</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>120</width>
               <height>130</height>
              </size>
             </property>
            </widget>
           </item>
           <item row="0" column="2">
            <widget class="LED" name="widget_5" native="true">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="minimumSize">
              <size>
               <width>120</width>
               <height>130</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>120</width>
               <height>130</height>
              </size>
             </property>
            </widget>
           </item>
           <item row="0" column="3">
            <widget class="LED" name="widget_6" native="true">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="minimumSize">
              <size>
               <width>120</width>
               <height>130</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>120</width>
               <height>130</height>
              </size>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QWidget" name="widget_2" native="true">
        <property name="styleSheet">
         <string notr="true">background-color: rgb(252, 233, 79);</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>DigitalClock</class>
   <extends>QWidget</extends>
   <header>digitalclock.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>LED</class>
   <extends>QWidget</extends>
   <header>led.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>AnimationButton</class>
   <extends>QWidget</extends>
   <header>animationbutton.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
